gen/dcos-config.yaml

```diff
@@ -489,30 +489,34 @@ package:
         ],
         "enabled": "{{ telemetry_enabled }}"
       }
-  - path: /etc_master/3dt_config_open.json
+  - path: /etc/dcos-3dt.env
+    content: |
+      DCOS_3DT_CONFIG_PATH=/opt/mesosphere/etc/dcos-3dt-config.json
+  - path: /etc_master/dcos-3dt-config.json
     content: |
       {
         "role": "master",
         "pull": true,
         "diagnostics-bundle-dir": "/var/lib/dcos/3dt/diag-bundles",
-        "endpoint-config": "/opt/mesosphere/etc/endpoints_config_open.json",
+        "endpoint-config": "/opt/mesosphere/etc/dcos-3dt-endpoint-config.json",
+        "no-unix-socket": true,
```

Comment:
we have 4 variants of 3dt listeners: OSS master, OSS agent, EE master and EE agent.
3 out of 4 are using unix socket by default, only OSS master uses TCP port for now. In future we are planning to switch to unix socket for OSS master and it will be as easy as just removing this line.